extensions/wikia/PortableInfobox/controllers/ApiQueryPortableInfobox.class.php

```diff
@@ -38,6 +38,14 @@ protected function runOnPageSet( ApiPageSet $pageSet ) {
 					$pageSet->getResult()->addValue(
 						[ 'query', 'pages', $id, 'infoboxes', $count ], 'sourcelabels', $sl
 					);
+
+					// TODO is this even needed? Seems to work without this code
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)